packages/amplify-e2e-tests/custom-resources/custom-cdk-stack-with-storage.ts

```diff
@@ -26,7 +26,6 @@ export class cdkStack extends cdk.Stack {
 
     /* Example 2: Adding IAM role to the custom stack */
     const role = new iam.Role(this, 'CustomRole', {
-      roleName: cdk.Fn.join('-', ['custom-cdk-generated-custom-role-test', cdk.Fn.ref('env')]), // For name uniqueness
```

Comment:
this not unique. Name is always `integtest` ...

Reply:
![image](https://user-images.githubusercontent.com/5849952/225768901-52636f94-a072-4c4b-9c0a-de349a6a3fac.png)